samples/auth-sample/auth-provider/src/main/resources/microservices/hello/hello.yaml

```diff
@@ -0,0 +1,55 @@
+swagger: '2.0'
```

Comment:
no need to provide schema, we can use codefirst rule.

Reply:
yes i will delete